### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by yosaddis**
#In this request I added the following features
- Added all images necessary for desktop version.
- Modify the index.html for desktop version of the website.
- Modify the CSS file for desktop resolution of 768px.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @yosaddis , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
image


## Invalid Code Review Request

You have submitted a project, that is **INVALID** ❌ ❌ 
- [ ] All linter files are missing 🔴 
- [ ] package.json file missing 🔴 
- [ ] Please refer to this l[lesson](https://github.com/microverseinc/linters-config/tree/master/html-css) and setup your linters correctly.



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -22,46 +22,55 @@ <h3 >
...
-		<p>
-			I can help you build a product , feature or website Look through some of my work and experience! If you like
-			what you see and have a project you need coded,
-			don’t hestiate to contact me.
-		</p>
+		<nav>
+			<ul class="header-nav-desktop">
+				<li>Portfolio</li>
+				<li>About</li>
+				<li>Contact</li>
```
Comment (iambenkis) [inline index.html]: - [ ] Good job so far :clap: , you made a beautiful project however, this button should be responsive , when you click on them , they should bring the user to the corresponding section :call_me_hand:. Kindly make sure you add that to improve the user experience.
### What you should do
```bash
<a href="#id_of_section"> Portfolio<a/>
```
Screenshot from 2023-01-20 15-01-39

File: index.html [hunk 2]
```diff
@@ -217,22 +213,22 @@ <h2 class="skill-name">Skills</h2>
...
+						<input class="email-input" type="email" name="email" placeholder="Email Address" required>
 					</li>
 					<li>
-						<textarea  type="text" name="message" maxlength="500" placeholder="Enter Text Here" cols="30" rows="10" required></textarea>
+						<textarea class="textarea-input" type="text" name="message" maxlength="500" placeholder="Enter Text Here" cols="30" rows="10" required></textarea>
 					</li>
 					<li>
-						<button type="submit">Get in touch</button>
+						<button class="form-submit-btn" type="submit">Get in touch</button>
 					</li>
```
Comment (iambenkis) [inline index.html:238]: - [ ] Kindly make sure that all your button have the interactive style (hover , active) to improve the user experience :call_me_hand: 
 
| Screenshot from 2023-01-20 15-08-30 |  Screenshot from 2023-01-20 15-08-15  |  Screenshot from 2023-01-20 15-07-57|


### What you could do 

```bash
button {
   cursor: pointer;
}

button:hover {
    color: xxx;
    background-color: yyyy;
}
````

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @yosaddis 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @yosaddis ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**